### ISSUE TITLE
Link - need support for title property

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents#7518]
### Is your feature request related to a problem?

This feature request is related to a11y requirement where user need to be informed about the link which will open in window/tab

### Describe the solution you'd like

Title property should be supported where we can provide extra info to user (in our case "info about link opening in new window").  This will help visual user as well as screen reader user.

For example:
```
<Link title="Link open in new window/tab">
  Link Text
</Link>
```

### Additional Context

Organization: {SuccessFactors}

### Declaration

- [X] I’m not disclosing any internal or sensitive information.
